src/Idris/REPL.idr

```diff
@@ -644,6 +644,63 @@ loadMainFile f
            [] => pure (FileLoaded f)
            _ => pure (ErrorsBuildingFile f errs)
 
+
+nfun : {auto c : Ref Ctxt Defs} ->
```

Comment:
As a local name `nfun` is probably fine but as a toplevel function I'd rather
have something a bit more descriptive. Maybe `replEval` as it seems to be
computing the semantics of `REPLEval`?